plugins/basic/python/tk_maya_basic/plugin_logging.py

```diff
@@ -41,15 +38,24 @@ def emit(self, record):
 
         :param record: Logging record to display in Maya script editor.
         """
-        # Give a standard format to the message.
-        msg = self.format(record)
 
-        # Display the message in Maya script editor in a thread safe manner.
+        # Give a standard format to the message:
+        #     Shotgun DEBUG: <message>
+        # or
+        #     Shotgun: <message>
+        # We use a simpler message format than the one used by the engine.
+        if record.levelno < logging.INFO:
+            msg = "Shotgun DEBUG: %s" % self.format(record)
```

Comment:
this is broken for multi line log entries so i am reverting back to the `fct = OpenMaya2.MGlobal.displayInfo` we have everywhere else.